src/components/navbar.jsx

```diff
@@ -1,8 +1,10 @@
+import { Link } from "react-router-dom";
+
 export const Navbar = () => {
     return(
         <nav className="nav space-around">
           <div> 
-              <a className="nav-header" href="/">The Sports Adda</a>
```

Comment:
You can replace anchor tag with link or button

Reply:
done that